src/main/java/frc/robot/Constants.java

```diff
@@ -31,6 +38,7 @@ public static class MotorConstants {
     public static final double NEO_FREE_SPEED_RPS = NEO_FREE_SPEED_RPM / 60;
     // MPS = (GearRatio * 2πr * RPM) / 60
     public static final int NEO550_FREE_SPEED_RPM = 11000;
+    public static final int NEO_VORTEX_CURRENT_LIMIT = 80;
```

Comment:
Where did this value come from? The [spark flex](https://docs.revrobotics.com/brushless/spark-flex/specs) isn't rated for more than 60A continuous